spec/routing/api/v1/ratings_routing_spec.rb

```diff
@@ -1,29 +1,33 @@
 require 'rails_helper'
 
-RSpec.describe Api::V1::RatingsController, type: :routing do
-  describe 'routing' do
-    it 'routes to #index' do
-      expect(get: '/api/v1/ratings').to route_to('api/v1/ratings#index')
-    end
+module Api
```

Comment:
what is the reason to wrap in modules?

Reply:
it turned out to be useless

Reply:
Module wraps are great places to have controllers, services, concerns, constants and any other code that, by having the same responsibility they should stay together)
namespacing is power) where it is of course really required)

Reply:
thanks)